### PR TITLE
Fixed SWIG go for cases when SWIG %import-s another package which is located in a subdirectory.

### PR DESCRIPTION
**Pull request by mishas**
Nothing much to add above the subject.

### CODE REVIEW

File: Examples/Makefile.in [hunk 1]
```diff
@@ -1819,14 +1819,24 @@ go: $(SRCDIR_SRCS)
 go_cpp: $(SRCDIR_SRCS)
 	$(SWIG) -go -c++ $(GOOPT) $(GOSWIGARG) $(SWIGOPT) -o $(ICXXSRCS) $(INTERFACEPATH)
 	if $(GO12) || $(GO13) || $(GOGCC); then \
-	  $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) $(SRCDIR_SRCS) $(SRCDIR_CXXSRCS) $(ICXXSRCS) $(INCLUDES); \
+	  if test -n "$(SRCDIR_CXXSRCS)$(SRCDIR_SRCS)"; then \
+	    $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) $(SRCDIR_CXXSRCS) $(SRCDIR_SRCS) $(INCLUDES); \
+	  fi; \
+	  ${foreach f,$(ICXXSRCS), \
+	    $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) -o ${addsuffix .o,${basename $f}} $f $(INCLUDES); \
```
Comment (ianlancetaylor) [inline Examples/Makefile.in]: The other function calls in this Makefile consistently use $(), not ${}.  Seems that this part may as well be consistent.

Comment (mishas) [inline Examples/Makefile.in]: Done.

**Comment by ianlancetaylor** [comment]
Thanks.  Now it's telling me that I can't automatically merge--can you update to master?

**Comment by mishas** [comment]
Merged (Didn't create a new merge CL, I just rebased everything).

**Comment by ianlancetaylor** [comment]
LGTM.  Thanks.

**Pull request merged**